packages/apollo-gateway/src/index.ts

```diff
@@ -285,15 +285,16 @@ export class ApolloGateway implements GraphQLService {
     ) {
       this.logger.debug('No change in service definitions since last check');
       return;
-    } else {
-      this.serviceDefinitions = result.serviceDefinitions;
+    }
+
+    if (previousSchema) {
```

Comment:
might we want a log line for like "started up w/ schema from service w/ ID `blah`" ?

Reply:
Resolved in 91a4e89